src/Form/Form.module.css

```diff
@@ -0,0 +1,20 @@
+form {
```

Comment:
I would suggest using classes wherever possible here.

I believe this CSS will affect _all_ `<form>` elements on the entire application, not just this one! CSS Modules does a fancy thing where it hashes css classes so they are unique per file, but it wouldn't do that with element selectors such as this or it would break.

You can even just update `form` to `.form` here and it would be sufficient. Remember to then reference it in `Form.js` like `<form className={classes.form}>`!

Reply:
Ahhh I'm still new to CSS Modules, so wasn't sure of the scope of these. Good to know!